client/src/layout/Header/Header.js

```diff
@@ -1,7 +1,32 @@
-import React from 'react';
+import React, { useState, useEffect } from 'react';
+import AppBar from '@mui/material/AppBar';
+import Toolbar from '@mui/material/Toolbar';
+import Container from '@mui/material/Container';
+import BurgerMenu from '../BurgerMenu/BurgerMenu';
+import TabletMenu from '../TabletMenu/TabletMenu';
+import LogoHeader from '../LogoHeader/LogoHeader';
+import HeaderSearchField from '../HeaderSearchField/HeaderSearchField';
 
 const Header = () => {
-  return <div></div>;
+  const [scrollY, setScrollY] = useState();
+
+  useEffect(() => document.addEventListener('scroll', () => setScrollY(window.scrollY)), []);
```

Comment:
could you add a delete of listener on unmounting?

Reply:
done